api-reference/30 Data Layer/PivotGridDataSource/1 Configuration/fields/format.md

```diff
@@ -162,5 +162,193 @@ See the [format](/api-reference/50%20Common/Object%20Structures/format '/Documen
 
 ---
 
+When being [exported to Excel](/Documentation/ApiReference/UI_Widgets/dxPivotGrid/Configuration/export/), the specified format is converted to an Excel format as follows:
```

Comment:
"When being [exported to Excel]" When what is being exported?

Reply:
Here we export formatting

Reply:
```suggestion
When a format is [exported to Excel](/Documentation/ApiReference/UI_Widgets/dxPivotGrid/Configuration/export/), it is converted to an Excel format as follows:
```